app/assets/javascripts/login_activity/LoginActivityPage.test.js

```diff
@@ -1,18 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import LoginActivityPage from './LoginActivityPage';
 import fetchMock from 'fetch-mock/es5/client';
+import LoginActivityPage from './LoginActivityPage';
+import loginActivityJson from './loginActivityJson.fixture';
 
-it('renders without crashing', () => {
+function mockFetch(props = {}) {
   fetchMock.restore();
-  fetchMock.get(
-    '/api/login_activity?created_at_or_before=1532105716&created_after=1529513716',
-    '[]'
```

Comment:
Without any data in the fixture here, most of the code in the component wasn't getting run in the test before, which is why the test didn't catch the bug.  So here I grabbed the response from my local development environment, and then have it use that which is a fast way to get a rough smoke test.